static/css/components/buttonGhost.less

```diff
@@ -3,6 +3,10 @@ a {
     padding: 5px 8px;
     border-radius: 5px;
     border: 1px solid @mid-grey;
+    display: block;
+    max-width: 87px;
+    margin: auto;
+    margin-right: 10px;
```

Comment:
```suggestion
    margin-left: auto;
```
This is short and keep's it right aligned.

Reply:
The size of button changes drastically if height of that top grey bar is kept the same. 

Following are the scenarios:

**same height**
![Screenshot from 2021-09-11 09-55-00](https://user-images.githubusercontent.com/54068868/132935829-bd9a6d33-5c1b-4120-afd9-e7bd024a646e.png)

**small change in height**
![Screenshot from 2021-09-11 09-55-39](https://user-images.githubusercontent.com/54068868/132935858-aeddddac-38fd-4928-9d20-2c067cf5c8dc.png)

**intermediate change in height**
![Screenshot from 2021-09-11 09-56-02](https://user-images.githubusercontent.com/54068868/132935861-64411a68-24ee-47ec-b33d-ead75d3b96f5.png)

Which one should I commit? Also, changed heart icon size .

Reply:
@rgrupesh I think the top one is the best to keep it the same height

Reply:
Okay. I'll make those changes.